mapping_definitions/deputy_daytime_phonenumbers_mapping.json

```diff
@@ -104,7 +104,7 @@
             "alias": "",
             "requires_transformation": "",
             "lookup_table": "",
-            "default_value": false,
+            "default_value": 0,
```

Comment:
This field is a boolean so though `0` is probably fine, `false` makes more sense